### ISSUE TITLE
Information missing on query result

### ISSUE THREAD

**Issue by duizer** [project-origin/wallet#34]
## Description

`QueryGranularCertificates` returns an array of :

```
message GranularCertificate {
    ProjectOrigin.Register.V1.FederatedStreamId FederatedId = 1;
    GranularCertificateType Type = 2;
    uint32 Quantity = 3;
    string GridArea = 4;
    google.protobuf.Timestamp Start = 5;
    google.protobuf.Timestamp End = 6;
    repeated Attribute Attributes = 7;
}

enum GranularCertificateType {
    CONSUMPTION = 1;
    PRODUCTION = 2;
}

message Attribute {
    string Key = 1;
    string Value = 2;
}
```

But the implementation only sets `FederatedId` and `Quantity`.

## Acceptance criteria
- Returns
  - time period (in `GranularCertificate`)
  - GridArea (as `GranularCertificate`)
  - CertificateType (as `GranularCertificate`)
  - AssetId (former GSRN) (as `Attribute`)
  - tech code  (as `Attribute`)
  - fuel code  (as `Attribute`)

## Solution plan
- [x] Migrate database
- [x] Write test
- [x] Implement code

Fixes #35

**Comment by MartinSchmidt** [comment]
For reference of how we group attributes: https://project-origin.github.io/registry/concept/granular-certificates/attributes.html#secondary-attributes 
The only difference is that GSRN is a danish term and is set to be removed as a primary attribute and moved to a secondary attributed named AssertID, more on this later.

**Comment by tnickelsen** [comment]
Fixed by #35
